### ISSUE TITLE
Manufacture power/motor/servo boards

### ISSUE THREAD

**Issue by jcwilliamson** [sourcebots/tasks#5]

**Comment by kierdavis** [comment]
@trickeydan What are the next steps here? Shall I go ahead and start getting orders placed for the boards we can currently afford?

**Comment by kierdavis** [comment]
[Updated calculations](https://docs.google.com/spreadsheets/d/1zE_gw8idO3K-fdvhHla8w_M4fcu3P_LFyxQ6oa0C2X4/edit#gid=350195810) suggest we could manufacture all power and motor boards right now, and need an extra £2000 to do the servo boards as well.

**Comment by trickeydan** [comment]
@kierdavis

Let's forward the updated info to Son.

I'll try and meet him ASAP to discuss next steps, but I strongly suspect it'll involve ordering PB and MB this week.

**Comment by trickeydan** [comment]
Have spoken to Son, we're good to order the first two boards.

**Comment by kierdavis** [comment]
Brilliant, I'll aim to get things moving this evening.

**Comment by kierdavis** [comment]
Somewhat blocked on #62 in case it reveals new information about how many boards we need to manufacture.

**Comment by kierdavis** [comment]
Manufacture files have been finalised, and order instructions have been put together and sent to ECS for purchasing.

**Comment by trickeydan** [comment]
All boards have been ordered now.

**Comment by kierdavis** [comment]
The supplier (Beta Layout) have provided quotes for the cost of PCB fabrication and assembly. Hopefully they will be sending quotes for the cost of the components very soon.

**Comment by kierdavis** [comment]
Component quotes are trickling through.

Additionally, they found our polarity diagrams unclear, so I'm redoing them.

**Comment by kierdavis** [comment]
This has now been postponed until further notice at ECS's decision.

**Comment by kierdavis** [comment]
This has been un-postponed and is now proceeding as normal.

**Comment by kierdavis** [comment]
For future reference, the files for manufacture are being tracked here: https://github.com/sourcebots/kit-manufacture-2019

**Comment by trickeydan** [comment]
~17~ 16 motor boards have been received.

New Issues: #205 #206 #207 #208

**Comment by kierdavis** [comment]
Uhhh, 99% sure we ordered 16 motor boards

https://drive.google.com/file/d/1qTpOzz7vgAyTHtFtUROAdJadeXhEfRJ_/view?usp=sharing

**Comment by jcwilliamson** [comment]
Dan says that you are correct, there were 16 boards.

**Comment by trickeydan** [comment]
17 Power Boards have been received

#209 #210 #211 #212 #213
